container/container_windows.go

```diff
@@ -8,6 +8,11 @@ import (
 	"path/filepath"
 
 	containertypes "github.com/docker/docker/api/types/container"
+	"github.com/docker/docker/pkg/system"
+)
+
+const (
+	containerInternalConfigsDirPath = `C:\ProgramData\Docker\internal\configs`
```

Comment:
My intention of my comnent was os.Getenv would be preferred over hardcoding 😅

ProgramData itself looks good